ci/start_init_test.sh

```diff
@@ -373,6 +373,43 @@ get_replica_count() {
 	echo "$replicaCount"
 }
 
+verify_delete_replica() {
+    old_replica_count=$(get_replica_count $CONTROLLER_IP)
+    echo "$old_replica_count"
+    curl -X "POST" http://$CONTROLLER_IP:9501/v1/delete | jq
```

Comment:
let us compare for required error message here like "Error deleting replica" etc.
Let us pass 'expectedMsg' and check it in output of curl request.